HW1/P6/P6.py

```diff
@@ -0,0 +1,62 @@
+__author__ = 'xiaowen'
+import findspark
+findspark.init('/home/xiaowen/spark')
+import pyspark
+from collections import Counter
+import numpy as np
+
+
+# build sparkcontext object
+sc = pyspark.SparkContext(appName="P6")
+
+# read data, convert it to RDD
+lines = sc.textFile('/home/xiaowen/cs205-homework/HW1/P6/pg100.txt')
+
+# split words
+wlist_before = lines.flatMap(lambda x:  x.split())
+
+# filter out words only contain numbers, only capitalized letters or only capitalized end with a period
+wlist = wlist_before.filter(lambda x: not(x.isdigit() or x.isupper() or (x[-1] == '.' and x[:-1].isupper())))
+
```

Comment:
Borrowing comments made on my code:
I believe you can just check if x.isupper() or x.isdigit()
"HELLO.".isupper() returns True, so the additional check for the period seems repetitive